### PR TITLE
Added an EventBanner component to display flyers and a description on the front page

### LINKED ISSUE #5: As a user, I want to be able to see flyers for upcoming events at a glance (on the home page)
This ticket pertains strictly to the Flyer.jsx component which should include the following:
Non-React Components:
- Actual image file to be rendered, passed in as a prop
React Components:
- A pop-up modal that gives a more in depth look at the event details, probably FlyerModal.jsx

### LINKED ISSUE #6: As a public relations chair, I want to be able to update the website with new flyers
This is a long term, nice-to-have, feature that would basically involve creating an external way for @swiessler to upload flyers and event details without having to spin up the dev environment. We want to make it as simple as possible. Some ideas are:
### Python hooks:
- Python script to hook into Google Drive and check for file changes
- Anytime an image file is uploaded, it will detect this and also check for a .txt file of the same name
- It will download both files and automatically generate a template code snippet that just adds a new Flyer.jsx component using the image file and the parsed out data from the .txt file
### Electron app:
- Electron (JavaScript for desktop apps) app that provides a simple UI for inputting a flyer and event details
- See last point in "Python hooks" section

### PR DESCRIPTION
**Pull request by carpenterd777**
Closes #6, Closes #5 

As a future idea, I think it might be neat to pass in a `Date()` as a prop to the `EventBanner` and have it stop rendering on the site once the date has passed.

Let me know what you all think!

### CODE REVIEW

File: sec.club/src/components/EventBanner/EventBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./EventBanner.scss";
+
+export default function EventBanner({source, title, desc}) {
```
Comment (dyc3) [inline sec.club/src/components/EventBanner/EventBanner.jsx]: I would change `source` to be something more descriptive like `flyerSource`

**Comment by carpenterd777** [comment]
This component currently does not include the modals discussed in #5 but this feature could be implemented in the future.

File: sec.club/src/components/EventBanner/EventBanner.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
+@import "../../variables.scss";
+
+
+
```
Comment (dyc3) [inline sec.club/src/components/EventBanner/EventBanner.scss]: just 1 newline please

File: sec.club/src/components/EventBanner/EventBanner.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    
+            .description {
+                font-size: 1.5em;
+                height: fit-content;
+                margin: 0 1.2em;
+
+            }
+        }
+
+
```
Comment (dyc3) [inline sec.club/src/components/EventBanner/EventBanner.scss]: remove these newlines

File: sec.club/src/components/EventBanner/EventBanner.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            <div className="container">
+                <img src={flyer.default} alt={`${flyerSource}`}/>
+                <div className="text-container">
+                    <span className="title">{title}</span>
+                    <span className="description">{desc}</span>
+                </div>
+            </div>
+        </div>
+    )
+
```
Comment (dyc3) [inline sec.club/src/components/EventBanner/EventBanner.jsx]: remove extra newline here

File: sec.club/src/components/EventBanner/EventBanner.scss [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        display: flex;  
+        border: $primary-color 1px solid;
+
+        @media (max-width: $sm-max) {
+            margin: auto;
+        }
+
+        img {
+            max-width: 50%;
+            max-height: 60vh;
```
Comment (dyc3) [inline sec.club/src/components/EventBanner/EventBanner.scss]: Don't use viewport height units for something like this, it's harder to work with than you'd think. Stick with `px`

Comment (dyc3) [reply sec.club/src/components/EventBanner/EventBanner.scss]: Actually `%` might be better?

**Pull request merged**